test/tests.py

```diff
@@ -841,12 +844,13 @@ def test(self):
                           'r:xz') as tf:
             tf.extractall(dir)
         if is_windows():
-            assert (dir / '_' / os.path.basename(self.filename)).is_file()
+            assert (dir.joinpath(*self.prefix)
```

Comment:
This spreading seems wrong. `self.prefix` is a string, meaning each character of the prefix will become a separate argument to `joinpath`.

Reply:
good catch, fixed